### PR TITLE
js exercise by ali

### PR DESCRIPTION
**Pull request by aliulgr13**
It needs to be examined.

### CODE REVIEW

**Comment by aliulgr13** [comment]
@wilgert  my homework is ok, do I need to change or add something?

File: Week1/homework/js-exercises/compareArrays.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict'
+const bestDefender = ["Van Dijk", "Ramos", "Söyüncü"];
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/compareArrays.js]: I would add the `s` at the end of the variable, to show that there are more elements in the variable.

So I would change it to: `bestDefenders`

File: Week1/homework/js-exercises/logLengtString.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+'use strict'
+let mySentence = "Programming is so interesting!";
+let x = mySentence.length;
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/logLengtString.js]: Nice, I would change `x` to a name that is more meaningfull, for example: `mySentenceLength`

File: Week1/homework/js-exercises/roundNumber.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+'use strict'
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let highestValue = Math.max(a, z);
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/roundNumber.js:6]: ah nice! Nice solution

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
Very nice! I only have some small comments on the homework:) If you can fix that then I will approve it.

File: Week1/homework/js-exercises/compareArrays.js (hunk 1 above)
Comment (aliulgr13) [reply Week1/homework/js-exercises/compareArrays.js]: > I would add the `s` at the end of the variable, to show that there are more elements in the variable.
> 
> So I would change it to: `bestDefenders`

yeah it is a quite logical : )

File: Week1/homework/js-exercises/logLengtString.js (hunk 2 above)
Comment (aliulgr13) [reply Week1/homework/js-exercises/logLengtString.js]: yeah I understand esp. when we are working on a big project, meaningfull name would be useful, thank you.

File: Week1/homework/js-exercises/roundNumber.js (hunk 3 above)
Comment (aliulgr13) [reply Week1/homework/js-exercises/roundNumber.js:6]: Thank you Marciano

**Review by wilgert — APPROVED** [review]

**Pull request closed without merging**